Makefile

```diff
@@ -1,13 +1,30 @@
+ifeq ($(OS),Windows_NT)
+SHELL:=cmd.exe
+BLANK:=
+/:=\$(BLANK)
```

Comment:
We're defining an make variable named `/` that you can use with `$/`. It will resolve to the correct slash direction based on OS. Unfortunately, `/:=\` doesn't work syntactically for makefiles, so this is a [workaround](https://stackoverflow.com/a/54733416/2456938)

Reply:
Interesting. This makes sense. Can we add this comment to the Makefile so the knowledge is transferable.

Reply:
Will do :)